clusters/app.ci/prow-monitoring/mixins/_prometheus/job_failures_config.libsonnet

```diff
@@ -40,6 +40,12 @@
       'periodic-ci-kubevirt-hyperconverged-cluster-operator-release-4.5-hco-e2e-nightly-bundle-release-4-5-azure4': {
         receiver: 'openshift-virtualization',
       },
+      'periodic-ci-operator-framework-operator-lifecycle-managment-rhoperator-metric-e2e-aws-olm-release-4.5-daily': {
```

Comment:
shouldn't these go next to the defs for master?

Reply:
ahh, ok